Support complex Glob pattern

suharshan opened the pull request:
### Purpose
Without quotes complex glob patterns like `test/{controllers,routes,models,modules}/**/*.{js,ts}` can not be parsed.

suharshan committed:
Add quotes to glob

scruplelesswizard commented on action.yml:
```diff
@@ -58,7 +58,7 @@ runs:
           EXCLUDE_GLOB="-exclude-glob=${{ inputs.exclude-glob }}"
```
Can you please add the quotes to the `exclude-glob` as well?

```suggestion
          EXCLUDE_GLOB="-exclude-glob='${{ inputs.exclude-glob }}'"
```

scruplelesswizard reviewed (commented):
@suharshan Thank you very much for your contribution! Just one nit that I've added a suggestion for.

suharshan committed:
Add quotes to exclude-glob

Co-authored-by: Jason Murray <15822260+chaosaffe@users.noreply.github.com>

suharshan commented:
> @suharshan Thank you very much for your contribution! Just one nit that I've added a suggestion for.

Thanks for the suggestion @chaosaffe and sorry for the slow turnaround time. Updated accordingly.
